lib/SQLiteReporter.cpp

```diff
@@ -293,7 +293,9 @@ void mull::SQLiteReporter::reportResults(const std::unique_ptr<Result> &result,
 
   sqlite3_close(database);
 
-  outs() << "Results can be found at '" << databasePath << "'\n";
+  outs() << "Results can be found at '" << databasePath << "'\n"
+         << "Use the following command to generate HTML report:\n"
+         << "REPORT=" << databasePath << " " << "make build\n";
```

Comment:
@AlexDenisov I have to say that this appears to be one of the most useful features. I would rather change the wording but definitely leave it here. It is super convenient to speed up the iteration process.